### DISCUSSION TITLE
Support Invoices with undefined amounts

### DISCUSSION THREAD

**Discussion by madeken** [btcpayserver/btcpayserver#2344] [Ideas / Feature Requests]
Many websites (e.g. exchanges) allow the creation of invoices with undefined amounts. For bitcoin this would appear as simply a bitcoin address, while on lightning would appear as an invoice with a 0 amount.

This is useful for websites with their own on-site balance systems, as well as "pick your own price" type thing, such as donations.

**Comment by pavlenex** [comment]
This is already supported through LNURL for 0-amount invoices, and for on-chain simply by displaying an address.
